nameable.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    "#{@nameable.correct_name}.capitalize()"
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    "#{@nameable.correct_name}.truncate(10)"
+  end
+end
```

Comment:
- [ ] This method should `trim` the word given if `characters` are more than 10 it should output `Maximilian` but currently I am getting `Maximilianus.truncate(10)` please address that